modules/040-control-plane-manager/docs/FAQ.md

```diff
@@ -521,6 +521,10 @@ When deciding on the appropriate threshold values, consider resources consumed b
 
 ## etc backup and restore
 
+### What is done automatically
+
+CronJob `kube-system/d8-etcd-backup` starts automatically at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` directory on a random control-plane node in the cluster.
```

Comment:
Not random. Result is saved on the master node where job performed.

Reply:
The job runs on a random master. More precisely on a less loaded master.